### PR TITLE
Towards BG-1150: Move kyc to checkout page

### PR DESCRIPTION
**Pull request by ap-justin**
NOTE: shipping this to staging would turn off current auto receipt sending (would be fixed in [BG-1142](https://linear.app/better-giving/issue/BG-1142/update-receipt-emailer)) as `kycEmail` (along with address) is no longer passed

## Explanation of the solution
* create summary step that shows summary and requires user to input name and email
image

* checking out shows the same summary (per design) + checkout form

## Instructions on making this work

- run `yarn` or `yarn install` to install npm dependencies
- run `yarn run test --watchAll` to verify all tests still pass
- (optional) run `yarn run build` to verify the build passes
- run `yarn start` to start the webapp
-

## UI changes for review

When major UI changes are introduced with a PR, please include links to URLS to compare or screenshots demonstrating the difference and notify on design changes

### CODE REVIEW

File: biome.json [hunk 1]
```diff
@@ -38,7 +38,7 @@
         "noPrecisionLoss": "off",
         "noSwitchDeclarations": "off",
         "noChildrenProp": "off",
-        "noUnusedVariables": "error"
+        "noUnusedVariables": "warn"
```
Comment (ap-justin) [inline biome.json:41]: sometimes erroneous code and unused code go in the same line and its hard to distinguish which is which.`warn` is underlined yellow and error is underline in red
image

File: src/components/KYCForm/types.ts [hunk 2]
```diff
@@ -1,22 +1,18 @@
-import { DonationRecipient, KYC } from "slices/donation";
+import { Country, OptionType } from "types/components";
 
-export type FormValues = KYC;
-
-export type OnDonation = {
```
Comment (ap-justin) [inline src/components/KYCForm/types.ts:5]: KYC form is now only used for `post-donation` receipts, only available in `My donations` page

**Review by SovereignAndrey — DISMISSED** [review]

**Comment by SovereignAndrey** [comment]
@ap-justin Can you please fix the merge conflicts with the recent DAF donation flow changes?

File: src/components/donation/Steps/Submit/DAFCheckout/DAFCheckout.tsx [hunk 3]
```diff
@@ -1,33 +1,18 @@
...
     <div className="grid content-start p-4 @md:p-8">
-      <BackBtn type="button" onClick={() => dispatch(setStep("splits"))} />
-      <Heading classes="my-4" />
-      <SplitSummary
-        classes="mb-4"
-        total={<Currency {...currency} amount={total} classes="text-gray-d2" />}
-        liquid={<Currency {...currency} amount={liq} classes="text-sm" />}
-        locked={<Currency {...currency} amount={locked} classes="text-sm" />}
-      />
+      <BackBtn onClick={() => dispatch(setStep("donate-form"))} />
```
Comment (ap-justin) [inline src/components/donation/Steps/Submit/DAFCheckout/DAFCheckout.tsx:15]: @SovereignAndrey , 

> @ap-justin Can you please fix the merge conflicts with the recent DAF donation flow changes?

made the flow for manual DAF to just like of stocks
1. donate form stocks
2. skip straight to checkout instructions

as `splits` | `donorInfo` are not used in checkout

**Review by SovereignAndrey — APPROVED** [review]

**Pull request merged**